FileIO/GmshIO/GMSHPolygonTree.cpp

```diff
@@ -86,7 +86,11 @@ void GMSHPolygonTree::insertPolyline (GeoLib::PolylineWithSegmentMarker * ply)
 			if (! ply->isSegmentMarked(k)) {
 				std::size_t seg_num(0);
 				GeoLib::Point *intersection_pnt(new GeoLib::Point);
```

Comment:
Not, this PR, but anyway: This line could possible introduce a memory leak. I'd feel more comfortable by using `std::unique_ptr` here and `release()`, when you `pnt_vec.push_back()`, because the big loop below is a bit big. :)